### PR TITLE
RD-1766-redesign-ci-workflow-523

### PR DESCRIPTION
**Pull request by yuvlar**

### CODE REVIEW

File: jenkins/build-pod.yaml [hunk 1]
```diff
@@ -2,25 +2,38 @@ apiVersion: v1
 kind: Pod
 spec:
   containers:
-  - name: rpmbuild
-    image: rpmbuild/centos7
+  - name: jnlp
```
Comment (Gelio) [inline jenkins/build-pod.yaml:5]: I see these are new changes, best if @warolv reviews this file

**Review by Gelio — APPROVED** [review]

File: jenkins/Jenkinsfile [hunk 2]
```diff
@@ -22,37 +22,55 @@ pipeline {
...
+              echo 'prepare test folder'
+              sh "mkdir -p ${WORKSPACE}/test && cp -rf ${WORKSPACE}/${STAGE_DIR}/. ${WORKSPACE}/test"
+              dir("${WORKSPACE}/test"){
+                echo 'Run static code analysis and code style check'
+                sh 'npm run lint'
 
-                echo "Build application"
-                sh 'npm run build'
+                echo 'Check TypeScript types'
+                sh 'npm run check-types'
```
Comment (yuvlar) [inline jenkins/Jenkinsfile:73]: @Gelio I see now that in the original 5.2.3-build jenkinsfile there was no 'npm run check-types'.
Should I remove it from this PR? it looks like it's failing the pipeline as well.

Comment (Gelio) [reply jenkins/Jenkinsfile:73]: Yup, good point. It was added recently and it is only supported for 6.0+. Let's remove it :slightly_smiling_face:

Comment (yuvlar) [reply jenkins/Jenkinsfile:73]: will do, thanks

**Review by Gelio — APPROVED** [review]

**Pull request merged**